src/frontend/components/form/isbnIsmnRegistrationForm/Preview.jsx

```diff
@@ -193,7 +193,7 @@ function Preview ({values, intl}) {
           {values.edition && (
             <ListComponent
               label={<FormattedMessage id="form.isbnIsmn.preview.edition" />}
-              value={values.edition}
+              value={values.edition + '.'}
```

Comment:
Use template literals in order to avoid type-related problems during string concatenation.